docs/the-new-architecture/landing-page.md

```diff
@@ -3,11 +3,26 @@ id: landing-page
 title: Introduction
 ---
 
-This section is the entry point for the new guide’s documentation.
-It contains some basic information about the New Architecture: its pillars, the version from which it has been made available and other very high-level information. Then, it should present the guide’s structure itself.
+Starting in version 0.68, React Native provides a New Architecture that improves several aspects of how JavaScript code and host platforms can integrate, such as:
 
-This section should also contains a sort of Driving guide based on the user use-cases:
+- synchronously measure and render React Native views at the same time as native views
```

Comment:
I noticed that the Architecture section on Fabric uses "host platform" and "host views", but I'm a little unsure when it's more appropriate to refer to "native code" instead (something I noticed STYLEGUIDE.md also discourages when possible).

Reply:
@cortinico any suggestion here?

Reply:
I saw this got removed so my input is probably not needed anymore 👍